index.bs

```diff
@@ -1741,6 +1767,7 @@ This is a [=static command=].
           setWindowRect: bool,
           userAgent: text,
           ? proxy: session.ProxyConfiguration,
+          ? userPromptHandler: session.UserPromptHandler,
```

Comment:
This also needs to be `unhandledPromptBehavior` similar to the above mentioned case.

Reply:
```suggestion
          ? unahandledPromptBehavior: session.UserPromptHandler,
```